reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -739,7 +741,7 @@ public void operationComplete(ChannelFuture future)  {
 		public void onComplete() {
 			if (ops.channel()
 			       .isActive()) {
-				ops.sendCloseNow(null, this);
+				ops.sendCloseNow(new CloseWebSocketFrame(), new WebSocketCloseStatus(-1, ""), this);
```

Comment:
My understanding was that emitting a close status of -1 leads to ISE in WebFlux. Couldn't this be a status of 1001 indicating normal completion?

Reply:
done